starlette_admin/actions.py

```diff
@@ -7,8 +7,10 @@ def action(
     name: str,
     text: str,
     confirmation: Optional[str] = None,
-    submit_btn_text: Optional[str] = _("Yes, Proceed"),
+    action_btn_class: Optional[str] = "btn-outline",
     submit_btn_class: Optional[str] = "btn-primary",
+    submit_btn_text: Optional[str] = _("Yes, Proceed"),
+    icon_class: Optional[str] = "fa-solid fa-plus",
```

Comment:
Can the `icon_class` be made optional? This would prevent any changes in the UI when upgrading.